shakenfist/images.py

```diff
@@ -30,6 +31,32 @@
     'ubuntu': image_resolver_ubuntu
 }
 
+IMAGE_FETCH_LOCK_TIMEOUT = 600   # TODO(andy):Should be linked to HTTP timeout?
```

Comment:
HTTP timeout is now exposed in config.parsed, so that would be possible if you wanted... Ultimately this is all about to get re-written to use a queue, so don't worry too hard about it.

Reply:
yep, leave to rewrite